packages/core/android/src/main/java/com/segmentanalyticsreactnative/AnalyticsReactNativeModule.kt

```diff
@@ -76,7 +76,7 @@ class AnalyticsReactNativeModule(reactContext: ReactApplicationContext) : ReactC
     }
 
     @ReactMethod
-    fun getContextInfo(config: ReadableMap, promise: Promise) {
+    fun getContextInfo(collectDeviceId: Boolean, promise: Promise) {
```

Comment:
We should keep the params as before so as to not differ from iOS. Also the config makes it more extendable.